<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabMealWidget</class>
 <widget class="QWidget" name="TabMealWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meal Tab</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="MealTabWidget" name="tab_meals"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="action_addday">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="list_meal"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MealTabWidget</class>
   <extends>QTabWidget</extends>
   <header>MealTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
